cmd/cloudflared/tunnel/configuration.go

```diff
@@ -348,6 +348,18 @@ func prepareTunnelConfig(
 	if err != nil {
 		return nil, nil, err
 	}
+	edgeBindAddr, err := parseConfigBindAddress(c.String("edge-bind-address"))
+	if err != nil {
+		return nil, nil, err
+	}
+	if err := testIPBindable(edgeBindAddr); err != nil {
+		return nil, nil, fmt.Errorf("invalid edge-bind-address %s: %v", edgeBindAddr, err)
+	}
+	edgeIPVersion, err = adjustIPVersionByBindAddress(edgeIPVersion, edgeBindAddr)
+	if err != nil {
+		// This is not a fatal error, we just overrode edgeIPVersion
+		log.Warn().Msgf("Overriding edge-ip-version to %s: %v", edgeIPVersion, err)
```

Comment:
We generally try to have static messages in log, and embed dynamic parts of the log in structured log fields. In that spirit, this message could have said "Overriding edge-ip-version", and then "edgeIPVersion": edgeIPVersion.

This comment also applies to anywhere else where logging is added.

Reply:
Looks like this is the only logging line added in this PR.